app/bundles/ApiBundle/Serializer/Driver/AnnotationDriver.php

```diff
@@ -12,17 +12,14 @@
 namespace Mautic\ApiBundle\Serializer\Driver;
 
 use JMS\Serializer\Metadata\ClassMetadata;
+use JMS\Serializer\Metadata\Driver\AnnotationDriver as BaseAnnotationDriver;
+use Metadata\ClassMetadata as BaseClassMetadata;
 
-/**
- * Class AnnotationDriver.
- */
-class AnnotationDriver extends \JMS\Serializer\Metadata\Driver\AnnotationDriver
+class AnnotationDriver extends BaseAnnotationDriver
 {
-    public function loadMetadataForClass(\ReflectionClass $class)
+    public function loadMetadataForClass(\ReflectionClass $class): ?BaseClassMetadata
```

Comment:
Why the question mark? Can it return null?

Reply:
Yes, the signature of the method on the interface has it returning null or the ClassMetadata object.

Reply:
Gotcha. Thanks Don!